packages/editor/src/store/utils/notice-builder.js

```diff
@@ -113,7 +108,7 @@ export function getNotificationArgumentsForSaveFail( data ) {
 	return [
 		noticeMessage,
 		{
-			id: SAVE_POST_NOTICE_ID,
```

Comment:
I think there was a reason that the save success and save error notices were using the same notice, though. 

Right now if you do the following:

* Save a post successfully
* Quickly block the save REST API request URL 
* Save quickly, see failure notice

You'll see the success notice remains together with the failure one:

![Screenshot 2024-12-30 at 13 10 31](https://github.com/user-attachments/assets/a4db00b1-315d-4b07-abe8-1d0ba8e864cc)

Haven't tested, but I believe that if you keep the same ID for both error and success, the new (failure) notice should remove the old (success) one properly.

Reply:
That's a good point. I'll update to use the same string value again.